packages/commerce-sdk-react/src/provider.tsx

```diff
@@ -74,7 +80,37 @@ const CommerceApiProvider = (props: CommerceApiProviderProps): ReactElement => {
         shopperProducts: new ShopperProducts(config),
         shopperPromotions: new ShopperPromotions(config),
         shopperSearch: new ShopperSearch(config)
-    }
+    })
+
+    useEffect(() => {
+        const newApiClients = {
+            shopperBaskets: new ShopperBaskets(config),
```

Comment:
Like how we've converted the `auth` object to use `useMemo`, we can also do the same here for the api clients.

Reply:
✅